src/strings/en-us.json

```diff
@@ -1696,6 +1697,8 @@
     "OptionLikes": "Likes",
     "OptionList": "List",
     "OptionLocked": "Locked",
+    "OptionLoginAttemptsBeforeLockout": "Determines how many incorrect password attempts can be made before lockout occurs.",
+    "OptionLoginAttemptsBeforeLockoutHelp":"0 inherits the old default of 3 for non admin and 5 for admin, -1 to disable lockout",
```

Comment:
There's a space missing after the semicolon, although I think it should parse fine.